from.js

```diff
@@ -65,6 +65,9 @@ class BlobDataItem {
     this.#start = options.start
     this.size = options.size
     this.lastModified = options.lastModified
+    this.originalSize = options.originalSize === undefined
```

Comment:
I personally try to treat null and undefined the same in the code, to avoid unexpected behaviour. This also lets us transition to the `??` operator when we drop support for Node.js 12.x, without changing the behaviour of the code.

```suggestion
    this.originalSize = options.originalSize == null
```

(`x == null` is the same as `x === undefined || x === null`)

Reply:
i know that, i also try to avoid null from time to times. and i can't wait until we can start using `??=`

but i wanted to avoid using == instead of === and breaking any compatibility with nullish stuff
also since there is no occurrences of null in the code, then i tought it would be fine to just check for undefined

could also just do `this.originalSize = options.originalSize || options.size`

Reply:
`options.originalSize || options.size` will use `options.size` if `options.originalSize` is `0` though, that's probably not what we want

Reply:
i thought about it too, but after commit [568b309](https://github.com/node-fetch/fetch-blob/pull/134/commits/568b3096b40ca2b1b489edde2f648caf84d9c527) then that might not be needed after all, cuz 0-sized blob will not be in the source anymore